README.md

```diff
@@ -187,7 +191,11 @@ After about 30-60 seconds the crons should have completed and you should have da
  browser 'compare snapshot' and a pop up appears to `Update Snapshot` 
 
 ## Jest Testing
-`docker exec -it webpack_watcher npm test` will run the test suite for the front-end React application. [Jest](https://jestjs.io/) is the testing framework used. More tests (including back-end tests) are planned for the future.
+Other front-end tests leverage the [Jest framework](https://jestjs.io/). 
+
+To run the Jest test suite, execute the command `docker exec -it webpack_watcher npm test`
+
+To update snapshots, execute `docker exec -it webpack_watcher npm update-snapshot`
```

Comment:
I've never done this before, but I just tried it, and `npm` doesn't know this command. Can you check this?

Maybe it needs to be something like this: `docker exec -it webpack_watcher npm test -- --update-snapshot`
But that actually runs the test suite, also; not sure if you intended for the command to do that.

Reply:
Looks like this was resolved by the latest commit. The use of `run-script` works for me.